testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -301,25 +298,35 @@ def __init__(self):
 
 
 class TestGROReaderNoConversion(BaseReaderTest):
-    def __init__(self, reference=None):
-        if reference is None:
-            reference = GRONoConversionReference()
-        super(TestGROReaderNoConversion, self).__init__(reference)
-        self.reader = self.ref.reader(self.ref.trajectory, convert_units=False)
-        self.reader.add_auxiliary('lowf', self.ref.aux_lowf,
-                                  dt=self.ref.aux_lowf_dt,
-                                  initial_time=0, time_selector=None)
-        self.reader.add_auxiliary('highf', self.ref.aux_highf,
-                                  dt=self.ref.aux_highf_dt,
-                                  initial_time=0, time_selector=None)
+    @staticmethod
+    @pytest.fixture()
+    def ref():
+        return GRONoConversionReference()
+
+    @staticmethod
+    @pytest.fixture()
+    def reader(ref):
```

Comment:
This is here because I have to set `convert_units` to `False`. This is required for the tests to pass.